### PR TITLE
Deconstruction_Recipes_Part_2

### PR DESCRIPTION
**Pull request by karols146**
Deconstruction_Recipes_Part_2

### CODE REVIEW

File: data/json/recipes/recipe_deconstruction.json [hunk 1]
```diff
@@ -276,223 +276,176 @@
...
-    "type" : "recipe",
+    "result": "gold_ear",
+    "type": "recipe",
+    "id_suffix": "uncraft",
+    "category": "CC_NONCRAFT",
+    "subcategory": "CSC_NONCRAFT",
+    "difficulty": 0,
+    "time": 500,
+    "reversible": true,
+    "qualities":[ {"id":"HAMMER","level":1,"amount":1} ],
```
Comment (Cyrano7) [inline data/json/recipes/recipe_deconstruction.json]: I'd include a space after the first colon on these lines.

Comment (Cyrano7) [inline data/json/recipes/recipe_deconstruction.json]: Also the ,"amount":1 can go.

Comment (karols146) [inline data/json/recipes/recipe_deconstruction.json]: I leave colons, because on last PR it just was like that.
I just don't want to mix style again.

Comment (mugling) [inline data/json/recipes/recipe_deconstruction.json]: Correct. The answer to that question has been given multiple times and the justification was sound. Increasing workload of both PR authors and reviewers is unhelpful.

**Comment by Cyrano7** [comment]
@mugling ? I thought that having a space to the right of the colon was the standard.

**Comment by mugling** [comment]
> @mugling ? I thought that having a space to the right of the colon was the standard.

@Cyrano7 Ah do you mean the `qualities` line in which case you are correct. I still think too much emphasis is being placed upon colon placement however if I came across as hostile I must appologise

**Comment by Cyrano7** [comment]
@mugling Ya normally a probably wouldn't have cared much about the colon spaces as I think putting to much focus on json style might scare away some entry level people. But since this is just a rewrite and clean up and not new content I figured I'd say something.

File: data/json/recipes/recipe_deconstruction.json [hunk 2]
```diff
@@ -284,7 +284,7 @@
     "difficulty": 0,
     "time": 500,
     "reversible": true,
-    "qualities": [ {"id":"HAMMER","level":1} ], 
+    "qualities": [ {"id": "HAMMER","level": 1} ],
```
Comment (karols146) [inline data/json/recipes/recipe_deconstruction.json]: Like that guys ?

Comment (Cyrano7) [inline data/json/recipes/recipe_deconstruction.json]: Up to @mugling

Comment (mugling) [inline data/json/recipes/recipe_deconstruction.json]: I suppose the canonical form would be:
`"qualities": [ { "id": "HAMMER","level": 1 } ]`

Comment (karols146) [inline data/json/recipes/recipe_deconstruction.json]: ok

**Comment by karols146** [comment]
@mugling All done should be OK now, in 1 party I change Qualities when finish with all.

**Pull request merged**